ignite/pkg/cosmosbuf/buf.go

```diff
@@ -96,7 +96,9 @@ func (b Buf) Update(ctx context.Context, modDir string, dependencies ...string)
 // Export runs the buf Export command for the files in the proto directory.
 func (b Buf) Export(ctx context.Context, protoDir, output string) error {
 	// Check if the proto directory is the Cosmos SDK one
-	if cosmosver.CosmosSDKModulePathPattern.MatchString(protoDir) {
+	// TODO(@julienrbrt): this whole custom handling can be deleted
+	// after https://github.com/cosmos/cosmos-sdk/pull/18993 in v29.
```

Comment:
The mentioned PR is merged already, should this code be changed? If so it could also be solved in a different issue.

Reply:
Yes, once v0.50.3 and v0.47.8 are released we can add a condition to skip that whole logic, and eventually delete it in v29.